igor_src/ExtractFeatures.h

```diff
@@ -0,0 +1,50 @@
+
+
+/* 
+ * File:   ExtractFeatures.h
+ * Author: alfaceor
+ *
+ */
+
+#ifndef EXTRACTFEATURES_H
+#define EXTRACTFEATURES_H
+#include <unordered_map>
+#include <vector>
+#include "CDR3SeqData.h"
+#include "Aligner.h"
+#include <sstream>
+#include "Utils.h"
+
+
+using namespace std;
+class ExtractFeatures {
```

Comment:
Could you add some documentation on the class? It doesn't have to be long
See for example the Alignment_Data class documentation:
```
/**
 * \class Alignment_data Aligner.h
 * \brief Stores information on the alignment of one genomic template against the target
 * \author Q.Marcou
 * \version 1.0
 *
 * Stores information on the alignment of one genomic template against the target.
 * It contains:
 * - the gene name
 * - the offset of the alignment (index on the target sequence on which the first letter of the FULL genomic template aligns (can be negative or lie outside the target)
 * - 5' and 3' offset positions of the best alignment first and last aligned nucleotide
 * - insertions : indices on the TARGET of inserted nucleotides
 * - deletions : indices on the GENOMIC TEMPLATE of deleted nucleotides
 * - alignment length
 * - list of mismatches (that lie event outside the best alignment to allow IGoR to know mismatch positions in advance while exploring different deletions numbers)
 * - the alignment score
 */
```